chain-indexer/src/infra/node/runtimes.rs

```
@@ -37,6 +40,8 @@ pub async fn make_block_details(
 ) -> Result<BlockDetails, SubxtNodeError> {
     if protocol_version.is_compatible(PROTOCOL_VERSION_000_012_000) {
         make_block_details_runtime_0_12(extrinsics, events, authorities).await
+    } else if protocol_version.is_compatible(PROTOCOL_VERSION_000_013_000) {
+        make_block_details_runtime_0_13(extrinsics, events, authorities).await
     } else {
         Err(SubxtNodeError::InvalidProtocolVersion(protocol_version))
```

adding error log also help production issue? depends on how we use the methods, it can fail silently in prod?

---

But if this fails, chain-indexer will crash and log the error.

---

ok 👍